src/tape/TapePlayer.cpp

```diff
@@ -23,14 +24,11 @@
 
 namespace fs = std::filesystem;
 
-TapePlayer::TapePlayer(const char* path)
+TapePlayer::TapePlayer(const char* path) : path(path)
 {
-	std::string pathYaml = fs::path(path).concat(YAML_EXTENSION).string();
-	std::string pathBin = fs::path(path).concat(BIN_EXTENSION).string();
-
-	playbackState = std::make_unique<PlaybackState>(pathBin.c_str());
+	playbackState = std::make_unique<PlaybackState>(getBinPath().c_str());
 
-	yamlRoot = YAML::LoadFile(pathYaml);
+	yamlRoot = YAML::LoadFile(getYamlPath());
```

Comment:
`getYamlPath`
I used only there I suppose. Is it worth it to encapsulate it in the method?

Reply:
`getBinPath()` is used in more places, so I did it just for the sake of completeness/symmetry.